README.md

```diff
@@ -1 +1,175 @@
-# MV-Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Leaderboard Microverse! Project <a name="about-project"></a>
+
+Leaderboard! project is a repository consisting of the following files:
+- HTML file 
+- CSS file 
+- JS files
+- Html,CSS and JS linters file
+
+
+The repository also contains files for setting up linters and validators.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+  <ul>
+    <li><a href="https://microverse.notion.site/HTML-CSS-Get-a-head-start-275eb85fd34b4416aa06ec635d69cdaf">HTML</a></li>
+    <li><a href="https://microverse.notion.site/HTML-CSS-Get-a-head-start-275eb85fd34b4416aa06ec635d69cdaf">CSS</a></li>
+       <li><a href="https://microverse.notion.site/HTML-CSS-Get-a-head-start-275eb85fd34b4416aa06ec635d69cdaf">JS</a></li>
+  </ul>
+
+
+
+### Key Features <a name="key-features"></a>
+## Leaderboard - [HTML/CSS & JavaScript Leaderboard project](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md)
+### Features Added:
+- Create a colection of name and number
+- Create a function to add a new task to the collection. from the api
+- Create a function to refresh the collection 
+- Display all tasks saved in the collection from the api.
+- CSS styles Added to the application to make it match [this](https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/awesome_books_core_elements.png) wireframe.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://benawi.github.io/Microverse-Awesome-books/)
```

Comment:
- [ OPTIONAL ] I noticed that your Live Demo links to the Awesome Books project and not to this leaderboard project. I will suggest that you include the right link to this project in your Readme so users are not misled when they visit your project repository :wink: